### DISCUSSION TITLE
[灌水干货]如何提升绳网等级？

### DISCUSSION THREAD

**Discussion by inkluv** [share121/inter-knot#1635] [其他]
![11111](https://github.com/user-attachments/assets/5e50a176-ecb0-4dfd-81b6-7cba4ddc29d8)

这个机制不太确定，就是
**Github的公开仓库数量=绳网等级**
_好细啊作者_

正题（灌水）：
如果你想升级，请你在github主页的 _右上角+号_
New repository，然后创建一个 **Public** 仓库，输入信息
然后可以选择添加Readme.md （md格式参考#1092）
里面输入一点正式的东西（可选）
然后创建就行了
如果你愿意，可以release里面塞点东西  [点我看示范](https://github.com/S11verWolf/TestLevel/releases/tag/%E7%AD%94%E8%BE%A9)
然后去绳网刷新就会发现升级了（可能

### 8.12更新：有大佬提出了天才的想法：
Fork别人的项目！

**Comment by Buer-Nahida** [comment]
《O p t i o n a l》

**Reply by inkluv** [reply]
听大佬说的 MPU 也可以awa

**Comment by inkluv** [comment]
听大佬说的 MPU 也可以awa

**Comment by capacitor1** [comment]
看出来了，真的水doge

**Comment by inkluv** [comment]
这个帖子最重要的是封面（

**Comment by Buer-Nahida** [comment]
奇怪的知识增加了，可选CPU
![x](https://i0.hdslb.com/bfs/emote/7145cbd5464b14a3f5cf544125a9976e44a13517.png)

**Comment by inkluv** [comment]
有大佬提出了天才的想法：
Fork别人的项目！
